pytest.ini

```diff
@@ -1,5 +1,5 @@
 [pytest]
-testpaths = benchmarks xain_fl
```

Comment:
Feel free to add a comment here to remove testpath `xain-fl`, once the benchmarks were moved out.

Reply:
makes sense! added

Reply:
Thanks.